recipes-bsp/firmware-imx/firmware-imx_8.23.bb

```diff
@@ -144,13 +151,15 @@ PACKAGES_DYNAMIC = "${PN}-vpu-* ${PN}-sdma-* ${PN}-easrc-* ${PN}-xcvr-* ${PN}-xu
 # is empty.
 # Therefore, we opt-out from producing -dev package here, since also for firmware
 # files it makes no sense.
-PACKAGES = "${PN} ${PN}-epdc ${PN}-hdmi"
+PACKAGES = "${PN} ${PN}-epdc ${PN}-hdmi ${PN}-vpu-imx8 ${PN}-vpu-imx95"
 
 FILES:${PN}-epdc = "${nonarch_base_libdir}/firmware/imx/epdc/"
 FILES:${PN}-hdmi = " \
     ${nonarch_base_libdir}/firmware/hdmitxfw.bin \
     ${nonarch_base_libdir}/firmware/hdmirxfw.bin \
     ${nonarch_base_libdir}/firmware/dpfw.bin \
 "
+FILES:${PN}-vpu-imx8 = "${nonarch_base_libdir}/firmware/amphion/vpu/*"
```

Comment:
Use `vpu-amphion` as suffix.

Reply:
Do you mean `FILES:${PN}-vpu-amphion-imx8`?

Reply:
I'd not use imx8 suffix as there is no restriction for NXP to make another SoC with Amphion VPU